packages/sku/lib/program/commands/translations/commands/compile/compile.action.js

```diff
@@ -0,0 +1,33 @@
+const { compile } = require('@vocab/core');
+const chalk = require('chalk');
+const { getResolvedVocabConfig } = require('../../helpers/translation-helpers');
+const { configureProject } = require('../../../../../utils/config-validators');
+
+const log = (message) => console.log(chalk.cyan(message));
+
+const compileAction = async ({ watch }) => {
+  await configureProject();
+  try {
+    const vocabConfigFromSkuConfig = await getResolvedVocabConfig({
+      translationSubCommand: 'compile',
+    });
+
+    log('Compiling translations...');
+
+    if (watch) {
+      log('Watching for changes to translations');
+    }
+
+    await compile({ watch }, vocabConfigFromSkuConfig);
+
+    if (!watch) {
+      log('Successfully compiled translations');
+    }
+  } catch (e) {
+    if (e) {
+      console.error(`Error running ${chalk.bold(`translations compile`)}:`, e);
```

Comment:
Is it safe to swallow the error here? Shouldn't it be re-thrown or in some way returned so the error code is set?

Reply:
The old `translations` command would `process.exit(1)` after catching any sub-command error, so maybe each sub-command needs its own one now.

Reply:
Actually, is it possible to hoist the `catch` to the top-level `translations` command, and then call `process.exit(1)` there.

Reply:
If the functions are async I can override the error behaviour of the command. let me check what I can do there. I don't think the translation action handler is aware of whats happening in the subcommands.